src/blurb/blurb.py

```diff
@@ -399,7 +399,7 @@ class BlurbError(RuntimeError):
     (This formatting will be inserted when rendering the final output.)
   * Lines longer than 76 characters will be wordwrapped.
       * In the final output, the first line will have
-        "- gh-issue-<gh-issue-number>: " inserted at the front,
+        "- :gh:`<gh-issue-number>`: " inserted at the front,
```

Comment:
Example output:

```
- gh-129248: The iOS test runner now strips the log prefix from each line
  output by the test suite.
```

Shall we match that?

```suggestion
        "- gh-<gh-issue-number>: " inserted at the front,
```

Because this is a docstring and let's not confuse with Sphinx roles?

Reply:
This PR changes the output to use Sphinx roles, though -- unless I've missed somewhere? 

The idea being we output the Sphinx roles here, rather than using a regular expression on the full text in the Sphinx build process.

Reply:
Got it, all good!